<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScaneLineWidget</class>
 <widget class="QWidget" name="ScaneLineWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="slidePointButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Przesuń punkt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addPointButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Dodaj punkt</string>
   </property>
  </widget>
  <widget class="QLabel" name="pointSizeLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Zmień rozmiar punktów</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removePointButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Usuń punkt</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="pointSizeSpinBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>390</y>
     <width>48</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <number>5</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QPushButton" name="clearButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Wyczyść</string>
   </property>
  </widget>
  <widget class="ScaneLineCanvas" name="scaneLineCanvas" native="true">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>531</width>
     <height>531</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="drawButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Rysuj</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScaneLineCanvas</class>
   <extends>QWidget</extends>
   <header>scanelinecanvas.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
